src/textual/widgets/_data_table.py

```diff
@@ -229,9 +233,16 @@ def get_cell_value(self, coordinate: Coordinate) -> CellType:
 
         Returns:
             The value of the cell.
+
+        Raises:
+            CellDoesntExist: If there is no cell with the given coordinate.
         """
         row, column = coordinate
-        return self.data[row][column]
+        try:
+            cell_value = self.data[row][column]
+        except KeyError:
+            raise CellDoesntExist(f"No cell exists at {coordinate!r}")
```

Comment:
This feels like the perfect place for you to teach me this: won't this raise the `CellDoesn'tExist` and show a traceback saying something "KeyError was raised and during handling of that exception, `CellDoesn'tExist` was raised"? Don't we need to add a `from None` here?

Reply:
I'd second a `from None` here.

Reply:
Added `from None`